daemon/cluster/convert/swarm.go

```diff
@@ -42,6 +42,7 @@ func SwarmFromGRPC(c swarmapi.Cluster) types.Swarm {
 			RootRotationInProgress: c.RootCA.RootRotation != nil,
 			DefaultAddrPool:        c.DefaultAddressPool,
 			SubnetSize:             c.SubnetSize,
+			DataPathPort:           c.VxlanUDPPort,
```

Comment:
If we can still rename this (in a follow-up) to `VXLANUDPPort`, that would probably be a good thing to do as well

Reply:
For exposing APIs and names for this feature as datapath port is good. But at the end of the day internally its going to be VXLAN  UDP port. Thats why I wanted to retain the same name at least inside swarmkit & libnetwork. Again, let me know if you think its still not a valid reason.

Reply:
Sorry, I meant `VXLANUDPPort` (capitalisation), to be consistent with the rest (and libnetwork)

Reply:
got it .

Reply:
hm... looks like we missed this one in the swarmkit change; I opened a PR; https://github.com/docker/swarmkit/pull/2783

Reply:
Let me see if we can get https://github.com/docker/swarmkit/pull/2783 merged 
(https://github.com/moby/moby/pull/38102/files#r234425632)